### PR TITLE
feat: upgrade to ml-testing-toolkit:v11.12.0

### PR DESCRIPTION
**Pull request by shashi165**
feat: https://github.com/mojaloop/project/issues/2098

### CODE REVIEW

**Review by vijayg10 — APPROVED** [review]

File: ml-testing-toolkit/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description:  ml-testing-toolkit Helm chart for Kubernetes
 name: ml-testing-toolkit
 version: 11.0.1
```
Comment (mdebarros) [inline ml-testing-toolkit/Chart.yaml:4]: Bump pls.

No magic here :D

**Review by mdebarros — CHANGES_REQUESTED** [review]
Minor change on bumping chart version.

You will also need to update the mojaloop/requirements.yml to reference the correct version.

File: ml-testing-toolkit/Chart.yaml (hunk 1 above)
Comment (shashi165) [reply ml-testing-toolkit/Chart.yaml:4]: > Bump pls.
> 
> No magic here :D

GOSH 🤣

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**